app/javascript/components/HighestRatedCryptid.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const HighestRatedCryptid = props => {
+  // let categories =['Aliens', 'Unicorns', 'Bigfoot', 'Lochness', 'Betty']
+  //
+  // let cryptids = categories.map((cryptid) => {
+  //   <li>{cryptid}</li>
+  // })
```

Comment:
Typically, I won't want to merge comments - I'm assuming this is the next bit you're going to work on so I'll keep it here, but definitely note that we want to leave comments out of PRs unless we intend to keep them for clarification!